recipes/GaitPy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "gaitpy" %}
+{% set version = "1.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/matt002/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3b0001220a2f1f19df638b3e26f9ab6838104decbf2dabf303aa71d6cdd29c42
+
+build:
+  skip: True  # [py!=36]
+  number: {{ build }}
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy ==1.13.3
+    - pandas ==0.20.3
+    - scipy ==1.2.0
+    - PyWavelets ==0.5.2
+    - scikit-learn ==0.21.3
+    - statsmodels ==0.8.0
+    - bokeh ==0.12.10
+    - dill ==0.2.7.1
+    - deepdish ==0.3.4
+  host:
+    - python
+    - pip
+    - numpy ==1.13.3
```

Comment:
Is there a reason for the strict pinning? Would `numpy >=1.13.3` etc. work instead?

Reply:
@npavlovikj Thank you for your feedback and edits. It is very important in my area of research that the methods/algorithms used are consistent going forward. I have experimented with unpinning certain dependencies, however I have received unit-test errors. Going forward for future iterations I will be diving into this issue more because I think it is better practice to have unpinned dependencies.